infra/resources/_modules/web_apps/cqrs-func.tf

```diff
@@ -156,7 +159,7 @@ module "cqrs_func" {
 
   tags = var.tags
 
-  application_insights_connection_string   = var.application_insights.connection_string
```

Comment:
why are you removing the connection string? the instrumentation key is deprecated and must not be used anymore